packages/hub/src/components/ui/EntityTab.tsx

```diff
@@ -0,0 +1,89 @@
+import { FC, ReactNode } from "react";
+import { IconProps, StyledTab } from "@quri/ui";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import clsx from "clsx";
+import React, { PropsWithChildren, createContext, useContext } from "react";
+
+type TabDivProps = {
+  name: string;
+  count?: number;
+  icon?: FC<IconProps>;
+  selected?: (pathname: string) => boolean;
+};
+
+type TabLinkProps = {
+  name: string;
+  href: string;
+  count?: number;
+  icon?: FC<IconProps>;
+  selected?: (pathname: string, href: string) => boolean;
+};
+
+type TabType = {
```

Comment:
This type is not necessary, Typescript infers it from `EntityTab`.

(it was different in `StyledTabType` because that type was more complicated, `FC & { ... }`.)